### DISCUSSION TITLE
how to make elements checked by default in multiselect

### DISCUSSION THREAD

**Discussion by mafin1799** [primefaces/community#746] [PrimeReact] [answered]
I have a list of options

they have a name, code and state, I can’t figure out how to apply the state of the elements, if they have selected: true then they are immediately selected

**Comment by gucal** [comment]
Hi,
Can you please write the problem more clearly? If you send a reproducer, it will help us understand the problem.

**Comment by mertsincan** [comment, marked as answer]
You can set your selected object or values into `value` prop. 

Best Regards,
